tools/pot/configs/README.md

```diff
@@ -34,21 +29,23 @@ This section contains only three parameters:
         "config": "./configs/examples/accuracy_checker/mobilenet_v2.yaml"
     }
 ```
-The main parameter is `"type"` which can take two possible options: `"accuracy_checher"` (default) and `"simplified"`,
-which specify the engine that is used for model inference and validation (if supported):
-- **Simplified mode** engine. This engine can be used only with `DefaultQuantization` algorithm to get fully quantized model 
-using a subset of images. It does not use the Accuracy Checker tool and annotation. To measure accuracy, you should implement 
-your own validation pipeline with OpenVINO API.  
-  - To run the simplified mode, define engine section similar to the example `mobilenetV2_tf_int8_simple_mode.json` file from the `<POT_DIR>/configs/examples/quantization/classification/` directory.
-- **Accuracy Checker** engine. It relies on the [Deep Learning Accuracy Validation Framework](@ref omz_tools_accuracy_checker) (Accuracy Checker) when inferencing DL models and working with datasets.
+The main parameter is `"type"` which can take two possible options: `"accuracy_checher"` (default), `"simplified"`, 
+ and `"data_free"` which specify the engine that is used for model inference and validation (if supported):
+- **Data-free mode** and **Simplified mode** engines. These engines can be used only with `DefaultQuantization` algorithm to get fully quantized model. They do not use the Accuracy Checker tool and annotation. In the case of this mode the following parameters are applicable:
+  - `"data_source"` (Optional for Data-free) Specifies path to directory​ where to calibration data is stored.
+  - `"layout"` - (Optional) Layout of input data. Supported values: [`"NCHW"`, `"NHWC"`, `"CHW"`, `"CWH"`]​.
+  - `"shape"` - (Optional) input shapes must be provided​ if model has dynamic shapes.
```

Comment:
As I know, `shape` is used to generate data for Data-free only

Reply:
@IndiraSalyahova, can you please confirm that Simplified mode does not need `shape` parameter? I am curious how we are going to deal with dynamic models in this case.

Reply:
@AlexKoff88 There is no need. As far as I remember, we discussed this and decided that if the model is dynamic, then we submit the picture without changes (if the height and width are not specified). If only height or width then exception. Ticket: 74653

Reply:
ok, thanks